rosbag2_storage/include/rosbag2_storage/storage_options.hpp

```diff
@@ -39,6 +39,11 @@ struct StorageOptions
   // A value of 0 indicates that bagfile splitting will not be used.
   uint64_t max_bagfile_duration = 0;
 
+  // The maximum number of bagfile splits to retain before the oldest bagfile is deleted.
+  // A value of 0 indicates that no deletion based on split count will occur.
+  // This feature is only available if either max_bagfile_size or max_bagfile_duration is set.
+  uint64_t max_splits = 0;
```

Comment:
```suggestion
  uint64_t max_bag_files = 0;
```
The naming is hard. I would propose to rename this parameter to `max_bag_files` to avoid confusion and multiple user questions, like:
- Will splits stop after reaching `max_splits`?
- Why is there one file when `max_splits = 1`? I would expect to have at least two files after the split.

Reply:
Updated.